<launch>
    <arg name="project" default="map_hokuyo" />

    <!--- State TF -->

    <param name="robot_description" command="$(find xacro)/xacro $(find map_hokuyo)/config/robot.urdf.xacro --inorder" />
    <node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher" respawn="true"></node>

    <!--- Nodes -->
    <node pkg="$(arg project)" type="StateTF.py" name="StateTF" output="screen" respawn="true"></node>
    <node pkg="$(arg project)" type="MapHokuyo.py" name="MapHokuyo" output="screen" respawn="true"></node>

    <!--- Run Rviz -->
    <node pkg="rviz" type="rviz" name="$(arg project)_rviz" args="-d $(find map_hokuyo)/config/rviz.rviz" />

</launch>